indra-common/src/main/kotlin/net/kyori/indra/data/License.kt

```diff
@@ -26,5 +26,6 @@ package net.kyori.indra.data
 data class License(
   val spdx: String?,
   val name: String,
-  val url: String
+  val url: String,
+  val bintray: String? = spdx
```

Comment:
@zml2008 does this work as I expect? or do we need to do something to allow groovy to not die?

Reply:
you'll need to add a `@JvmOverloads constructor` before the parameters